wikilegis/core/templatetags/wikilegis_votes.py

```diff
@@ -25,3 +25,23 @@ def voted_by_user(context, instance, vote):
             return ''
     else:
         return ''
+
+
+@register.assignment_tag(takes_context=True)
+def order_by_score(context, queryset):
+    list_score_id = []
+    for proposal in queryset:
+        ctype = ContentType.objects.get_for_model(proposal)
+        down_vote = models.UpDownVote.objects.filter(content_type=ctype,
```

Comment:
All models with `VoteCountMixin` has `downvote_count` attribute, to avoid unecessary database queries :)